config/routes.rb

```diff
@@ -1,56 +1,7 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+  get 'home/index'
```

Comment:
I don't think you meant to add an endpoint for `/home/index`. This line means that you want users to be able to visit `/home/index`. The root endpoint below is sufficient for mapping the root `/` endpoint to the `index` action of the HomeController. You can see what routes have been created for an app by running `rake routes`.